### PR TITLE
Add version information and render in front page

### PR DESCRIPTION
**Pull request by anshuldutt21**
Add a new file version information to keep record of the versions.
Render the version information in the about page of spdx.
Fixes for issue #93

### CODE REVIEW

**Comment by rtgdk** [comment]
@anshuldutt21 LGTM. Although, the version number for java is not correct, I think it is 2.1.8.
Can you also post a screenshot of the change, so that it is easier and quicker for people to review the changes?

**Comment by anshuldutt21** [comment]
@rtgdk Can you tell me where can I find the version of java tools and spdx? I can also write about it in the comments of the version file.

**Comment by rtgdk** [comment]
@anshuldutt21  Even, I found out about the version number from the last commit to that tar. But, you can play around with the tar using java : check https://github.com/spdx/tools

**Comment by anshuldutt21** [comment]
@rtgdk okay. Also, maybe we can start tagging spdx-online-tools and release the version starting from 1.0.

File: src/src/version.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+Version file for displaying the respective version of 
+the tools and online-tools repositories.
+"""
+
+spdx_version = '1.0'
+
+"""
+Visit https://github.com/spdx/tools/releases to know about the tools releases.
+"""
+java_tools_version = '2.1.20'
```
Comment (rtgdk) [inline src/src/version.py]: The current tar is not updated with the latest spdx tools release. I think it is 2.1.8 instead (check the last commit on the tar)

**Comment by anshuldutt21** [comment]
@rtgdk Is it fine now? Please review.

File: src/app/views.py [hunk 2]
```diff
@@ -78,7 +80,10 @@ def about(request):
     """ View for about
     returns about.html template
     """
-    context_dict={}
+    context_dict={
+                'spdx_version':spdx_version,
```
Comment (rtgdk) [inline src/app/views.py]: Change this to `spdx_online_tools_version`

Comment (rtgdk) [reply src/app/views.py]: spdx_version may be confused with with spdx specification version

**Comment by rtgdk** [comment]
@anshuldutt21 LGTM. I have suggested one more change. Need approval from @goneall @techytushar

File: src/app/views.py (hunk 2 above)
Comment (goneall) [reply src/app/views.py]: Agree with changing to online tools version

**Comment by anshuldutt21** [comment]
Please review @rtgdk

**Comment by anshuldutt21** [comment]
Please review @rtgdk

**Review by goneall — APPROVED** [review]

**Pull request merged**